File: latihan b/php/TimSepakbola.php
<?php

class TimSepakbola{

    //atribut private
    private $nama_tim = "";
    private $negara = "";
    private $tahun = "";
    private $pelatih = "";
    private $pemain = "";

    public function __construct($nama_tim = "", $negara = "", $tahun = "", $pelatih = "", $pemain = "")
    {
        $this->nama_tim = $nama_tim;
        $this->negara = $negara;
        $this->tahun = $tahun;
        $this->pelatih = $pelatih;
        $this->pemain = $pemain;
    }

    public function setNamaTim($nama_tim){
        $this->nama_tim = $nama_tim;
    }

    public function getNamaTim(){
        return $this->nama_tim;
    }

    public function setnegara($negara){
        $this->negara = $negara;
    }

    public function getnegara(){
        return $this->negara;
    }

    public function settahun($tahun){
        $this->tahun = $tahun;
    }

    public function gettahun(){
        return $this->tahun;
    }

    public function setpelatih($pelatih){
        $this->pelatih = $pelatih;
    }

    public function getpelatih(){
        return $this->pelatih;
    }

    public function setpemain($pemain){
        $this->pemain = $pemain;
    }

    public function getpemain(){
        return $this->pemain;
    }

}
?>

File: latihan b/python/TimSepakbola.py
class TimSepakbola():
    #private atribut 
    __namatim = ""
    __negara = ""
    __tahun = 0
    __pelatih = ""
    __pemain = ""

    #constructor
    def __init__(self, namatim = "", negara = "", tahun = 0, pelatih = "", pemain = ""):
        self.__namatim = namatim
        self.__negara = negara
        self.__tahun = tahun
        self.__pelatih = pelatih
        self.__pemain = pemain
    
    def setNamaTim(self, namatim):
        self.__namatim = namatim
    
    def getNamaTim(self):
        return self.__namatim
    
    def setNegara(self, negara):
        self.__negara = negara
    
    def getNegara(self):
        return self.__negara

    def setTahun(self, tahun):
        self.__tahun = tahun
    
    def getTahun(self):
        return self.__tahun

    def setPelatih(self, pelatih):
        self.__pelatih = pelatih
    
    def getPelatih(self):
        return self.__pelatih
    
    def setPemain(self, pemain):
        self.__pemain = pemain
    
    def getPemain(self):
        return self.__pemain